frontend/src/main/scala/akka/viz/frontend/FrontendApp.scala

```diff
@@ -2,8 +2,11 @@ package akka.viz.frontend
 
 import akka.viz.frontend.components.{OnOffPanel, MessagesPanel, MessageFilter, ActorSelector}
 import akka.viz.protocol._
+import org.scalajs.dom.ext.KeyCode
 import org.scalajs.dom.html.Input
-import org.scalajs.dom.{onclick => _, raw => _, _}
+import org.scalajs.dom.raw.{HTMLInputElement, KeyboardEvent, MessageEvent}
+import org.scalajs.dom.document
```

Comment:
![Codacy](https://www.codacy.com/versioned/images/favicon.png) Issue found: [Use brackets to group imports from the same package](https://www.codacy.com/app/lustefaniak/akka-viz/file/2004063021/issues/source?bid=3121282&fileBranchId=3122179#l8)